### PR TITLE
Replace non-free landscape images.

### PR DESCRIPTION
**Pull request by quyykk**
This replaces two landscapes images that were using a non-free license (the Pixabay License). Simple as that.

High DPI: https://github.com/endless-sky/endless-sky-high-dpi/pull/296

### CODE REVIEW

**Comment by Amazinite** [comment]
Would this mean also reverting https://github.com/endless-sky/endless-sky/pull/7033?

**Comment by quyykk** [comment]
Currently we're only using various CC license artwork but nothing precludes including artwork licensed under something else than CC, if that license still considered "free". So I guess we can just keep it for future-proofing.

File: copyright
Comment (Amazinite) [inline copyright:717]: Where do people keep grabbing this image? This looks like the same exact place as space2.
space2
I'd prefer if we had something different.

Comment (quyykk) [reply copyright:717]: whoops my bad

File: copyright
Comment (Amazinite) [inline copyright:711]: Station15 is used on an Efreti station, and I suppose the new image fits, but station28 is used on an abandoned Korath station. The older image at least has a dark, grimy appearance to it that might be fitting of an abandoned station, but this new one looks far to clean and bright.

**Review by Pointedstick — APPROVED** [review]

**Comment by Pointedstick** [comment]
Reviewing on technical merits; no comment on aesthetics. I'll leave that to others.

**Comment by oo13** [comment]
Unchanging README.md causes no harm, and probably some content creators will post a PR that has artworks published by "permissive" license, such as Pixabay License, CC-NC, or CC-ND.
Anyway, the package maintainers should check "copyright" file when they would release each release.

**Pull request merged**
